tools/monitoring/main.go

```diff
@@ -25,25 +25,40 @@ import (
 	"os"
 
 	"github.com/knative/test-infra/tools/monitoring/config"
+	"github.com/knative/test-infra/tools/monitoring/mail"
 	"github.com/knative/test-infra/tools/monitoring/mysql"
 )
 
-var dbConfig mysql.DBConfig
+var (
+	dbConfig   mysql.DBConfig
+	mailConfig *mail.Config
+
+	alertEmailRecipients = []string{"knative-productivity-oncall@googlegroups.com"}
+)
 
 const (
-	yamlURL              = "https://raw.githubusercontent.com/knative/test-infra/master/tools/monitoring/sample.yaml"
-	dbUserSecretFile     = "/secrets/cloudsql/monitoringdb/username"
-	dbPasswordSecretFile = "/secrets/cloudsql/monitoringdb/password"
+	yamlURL = "https://raw.githubusercontent.com/knative/test-infra/master/tools/monitoring/sample.yaml"
 )
 
 func main() {
 	var err error
 
 	dbName := flag.String("database-name", "", "The monitoring database name")
 	dbInst := flag.String("database-instance", "", "The monitoring CloudSQL instance connection name")
+
+	dbUserSecretFile := flag.String("database-user", "/secrets/cloudsql/monitoringdb/username", "Database user secret file")
```

Comment:
[nit] the var names feels too long :D

Reply:
Thanks for pointing out good golang practice. Updated it to use a shorter name.